docs/data/sdks/javascript/index.md

```diff
@@ -5,17 +5,16 @@ icon: simple/javascript
 ---
 
 <!-- vale off-->
-[![npm version](https://badge.fury.io/js/amplitude-js.svg)](https://badge.fury.io/js/amplitude-js)
+[![npm version](https://badge.fury.io/js/amplitude-js.svg)](https://www.npmjs.com/package/amplitude-js)
```

Comment:
where did you get the new one? I got the link the same as our old one. 
![image](https://user-images.githubusercontent.com/31029607/227058971-33182ff9-738a-4e5b-9278-984d2a458cc6.png)

Reply:
This should be updated to use `img.shields.io`.

Reply:
Updated.